standards.iso.org/iso/19115/resources/transforms/ISO19139/mapping/core.xsl

```diff
@@ -433,7 +433,7 @@
       </xsl:element>
     </xsl:element>
   </xsl:template>
-  <xsl:template match="gmi:objectiveOccurance" mode="from19139to19115-3">
+  <xsl:template match="gmi:objectiveOccurrence" mode="from19139to19115-3">
```

Comment:
While `objectiveOccurrence` is the correct spelling, I think that the spelling used in ISO 19115-3 was `objectiveOccurence` with only one "r". So this fix would need to only replace the "a" into "e" without adding "r".

Reply:
Reverted to `gmi:objectiveOccurence`. Thanks @desruisseaux !!!